examples/all-clusters-app/esp32/main/CHIPDeviceManager.cpp

```diff
@@ -99,6 +99,13 @@ CHIP_ERROR CHIPDeviceManager::Init(CHIPDeviceManagerCallbacks * cb)
 void MatterPostAttributeChangeCallback(const chip::app::ConcreteAttributePath & path, uint8_t mask, uint8_t type, uint16_t size,
                                        uint8_t * value)
 {
+    TaskHandle_t task = xTaskGetCurrentTaskHandle();
+    const char * name = pcTaskGetName(task);
+    if (!strcmp(name, "CHIP"))
```

Comment:
Should it be `if(strcmp(name, "CHIP"))`? I got the error logs when I ran all-clusters-app on esp32. Looks like this check is incorrect.

Reply:
Yes. Looks like :-)

Reply:
No, the check is correct.  We're logging when the thread name is not "CHIP".

Reply:
Er, yes, this is backwards.  Will create PR to fix as soon as ToT is not failing CI.

Reply:
https://github.com/project-chip/connectedhomeip/pull/15187